packages/ui-kit-components/src/icons/index.ts

```diff
@@ -13,16 +13,21 @@ export { ReactComponent as ChevronUp } from './images/ChevronUpIcon.svg';
 export { ReactComponent as Close } from './images/CloseIcon.svg';
 export { ReactComponent as Cog } from './images/CogIcon.svg';
 export { ReactComponent as Dots } from './images/DotsIcon.svg';
+export { ReactComponent as Download } from './images/DownloadIcon.svg';
 export { ReactComponent as Edit } from './images/EditIcon.svg';
+export { ReactComponent as Eye } from './images/EyeIcon.svg';
 export { ReactComponent as Filter } from './images/FilterIcon.svg';
 export { ReactComponent as Groups } from './images/GroupsIcon.svg';
 export { ReactComponent as Info } from './images/InfoIcon.svg';
 export { ReactComponent as Lock } from './images/LockIcon.svg';
+export { ReactComponent as More } from './images/MoreIcon.svg';
```

Comment:
I think the `More` icon is a duplicate of `Dots` and the `Eye` icon is a duplicate of `View`.

Reply:
They have been replaced by view and dots icons.